apis/python/CMakeLists.txt

```diff
@@ -64,14 +64,17 @@ target_compile_definitions(
         VERSION_INFO=${PROJECT_VERSION}
 )
 
-if (APPLE)
-  set_target_properties(${VSPY_TARGET_NAME} PROPERTIES INSTALL_RPATH "@loader_path/lib")
-elseif(UNIX)
-  set_target_properties(${VSPY_TARGET_NAME} PROPERTIES INSTALL_RPATH "\$ORIGIN/lib")
-endif()
-
 # Install the extension module
 install(TARGETS ${VSPY_TARGET_NAME} LIBRARY DESTINATION .)
 
-# Install libtiledb so that we can repackage it with the extension
-install(IMPORTED_RUNTIME_ARTIFACTS TileDB::tiledb_shared DESTINATION lib)
+if (TILEDB_DOWNLOADED)
+  message(STATUS "Adding libtiledb into install group")
+  # Install libtiledb so that we can repackage it with the extension
+  install(IMPORTED_RUNTIME_ARTIFACTS TileDB::tiledb_shared DESTINATION lib)
```

Comment:
Should this be `lib` or should we use `GNUInstallDirs` to get the library folder name? Or does it not matter because we are setting the RPATH below?

Reply:
Well, good question. Right now it is only used for wheel packaging, and while packaging wheels I believe that the install prefix is overridden, so it should have no effect. If you want this to 100% work even if installed into system then yes, I believe that we should change it. But I don't think that anyone uses it that way.

Reply:
Fixed, what about that first install? `install(STUFF .)`